cluster-install/install.sh

```diff
@@ -203,6 +207,17 @@ set_log_level() {
     [ "$LOG_LEVEL" == "fatal" ] && LOG_LEVEL="$__FATAL"
 }
 
+ingest_config_string() {
+    local -r config_str="$1"
+    [ -z "$config_str" ] && fatal "no config string given"
+    local -r tmp_env="$__TMP_ENV"
+
+    echo "$config_str" > "$tmp_env"
+    # shellcheck disable=SC1090
+    source "$tmp_env"
+    rm -f "$tmp_env" &> /dev/null
+}
```

Comment:
This is really the only place that new temporary env file is used. There's probably a way to do what this function is doing without creating a temporary file, but no matter what bash syntax I threw at the script, I couldn't really get it working. Decided not to waste too much time and just use what works for now.